uap_tracker/background_subtractor_factory.py

```diff
@@ -6,18 +6,29 @@
 class BackgroundSubtractorFactory():
 
     @staticmethod
-    def create(type, sensitivity=2):
+    def create(type, sensitivity):
         if type == 'KNN':
             # defaults: samples:2, dist2Threshold:400.0, history: 500
             background_subtractor = cv2.createBackgroundSubtractorKNN()
+
             # samples = background_subtractor.getkNNSamples()
             # dist_2_threshold = background_subtractor.getDist2Threshold()
             # history = background_subtractor.getHistory()
             # print(f'samples:{samples}, dist2Threshold:{dist_2_threshold}, history, {history}')
-            # MG TODO If clause here to handle the sensitivity parameter e.g. how aggressive do we want the subtraction to be
+
             background_subtractor.setHistory(1)  # large gets many detections
-            background_subtractor.setkNNSamples(2)  # 1 doesn't detect small object
-            background_subtractor.setDist2Threshold(5000)  # small gets many detections, large misses small movements
+
+            if sensitivity == 1:  # Detects small, medium and large objects
```

Comment:
@Micky-G  I'd probably have ordered these differently, I tend to think of high sensitivity as being very sensitive to small changes.

Reply:
@UAPPaul That is what its suppose to be to be fair i.e.:

DETECTION_SENSITIVITY_HIGH = 1
DETECTION_SENSITIVITY_NORMAL = 2
DETECTION_SENSITIVITY_LOW = 3

Have I misunderstood those parameters? I'll try and take another look at the documentation today.

Reply:
@UAPPaul Would you let me know what is wrong please mate? I have tested the sensitivity again and it appears to be working as I would expect.